alica_tests/src/test/test_alica_logger.cpp

```diff
@@ -2,25 +2,111 @@
 
 #include <gtest/gtest.h>
 
-namespace alica
+namespace alica::test
```

Comment:
Maybe change this when PRing the change for everything? idk, seems weird to have just one with this.

Reply:
didn't get you

Reply:
the namespace change would be nice to have in all test files for `alica_tests`

Reply:
yes, will do that eventually. Right now, we are going through one test file at a time & rewriting/improving them